libnopegl/src/node_fontface.c

```diff
@@ -29,6 +29,8 @@ static const struct node_param fontface_params[] = {
     {"path",  NGLI_PARAM_TYPE_STR, OFFSET(path),
               .flags=NGLI_PARAM_FLAG_NON_NULL | NGLI_PARAM_FLAG_FILEPATH,
               .desc=NGLI_DOCSTRING("path to the font file")},
+    {"index", NGLI_PARAM_TYPE_I32, OFFSET(index),
+              .desc=NGLI_DOCSTRING("index of the face in the font file")},
```

Comment:
We should probably mention that the index is related to freetype `face_index` (https://freetype.org/freetype2/docs/reference/ft2-face_creation.html#ft_open_face).

Reply:
I'd rather not, freetype is an implementation detail. FT doc says "Bits 0-15 are the index of the face in the font file (starting with value 0)", so I believe using the description "index of the face in the font file" is consistent with that